ui/src/app/settings/reducers/config.js

```diff
@@ -0,0 +1,25 @@
+import produce from "immer";
+
+const config = produce(
```

Comment:
Ah, looks like you and @steverydz have overlapped a bit here. You guys may need to coordinate to get this landed.